packages/coinjoin/tests/client/inputRegistration.test.ts

```diff
@@ -342,6 +342,7 @@ describe('inputRegistration', () => {
         expect(spy.mock.calls.length).toBeGreaterThan(0);
 
         // 2. wait for confirmation interval to resolve
+        // eslint-disable-next-line jest/valid-expect-in-promise
```

Comment:
Not my code, but it seems that `input.clearConfirmationInterval()` must be called right after `input.getConfirmationInterval()` in order to resolve it, therefore it can't be awaited. How 'bout something like this? Will it satisfy the eslint rule?

```typescript
        // 2. wait for confirmation interval to resolve
        const promises = Promise.all(
            response.inputs.map(input => input.getConfirmationInterval()?.promise),
        );

        // 1. abort confirmation interval
        response.inputs.forEach(input => input.clearConfirmationInterval());

        const res = await promises;
        res.forEach(input => {
            expect(input?.getConfirmationInterval()).toBeUndefined();
        });
```